test/data/check-uk-visa-responses-and-expected-results.yml

```diff
@@ -650,7 +650,7 @@
   - kuwait
   - study
   - six_months_or_less
-  :next_node: :outcome_visit_waiver
+  :next_node: :outcome_study_waiver
```

Comment:
It looks as if this change was the result of this commit: 5443c5429c9ba9522b87e5f4d062e6d43cb99fdd which was part of #2524. I'm therefore happy to include it.